wtxmgr/tx.go

```diff
@@ -174,7 +174,7 @@ func (s *Store) InsertAddrTxOut(ns walletdb.ReadWriteBucket, txOut *AddrTxOutput
 	} else {
 		k := canonicalOutPoint(&txOut.TxId, txOut.Index)
 		if !txOut.IsBlue {
```

Comment:
Besides coinbase transaction, Ordinary transactions in red block is also legal.

Reply:
ok